UE/Application/States/ConnectedState.cpp

```diff
@@ -31,6 +39,7 @@ void ConnectedState::handleCallRequest(common::PhoneNumber fromPhoneNumber)
     };
     context.user.setupIncomingCallButtons(acceptButtonCallback, rejectButtonCallback);
 
```

Comment:
delete blank line
```suggestion
```

Reply:
I think it is fine like that, as it visually separates the _button_ and _timer_ portions of the function ;)